test/unit_tests/Makefile

```diff
@@ -9,7 +9,7 @@ SRCS :=  \
 	$(LIB_DIR)/thirdparty/kissfft/kiss_fft.c \
 	$(LIB_DIR)/thirdparty/kissfft/kiss_fftnd.c \
 	$(LIB_DIR)/blob_detection.cpp \
-    ${LIB_DIR}/filtering.cpp
+    ${LIB_DIR}/filtering.cpp \
```

Comment:
I'll revert this change and fix it separately. Thanks a lot for pointing the issue!